start-site/src/main/resources/application.yml

```diff
@@ -203,6 +203,16 @@ initializr:
           artifactId: spring-boot
           description: Support for compiling Spring applications to native executables using the GraalVM native-image compiler.
           starter: false
+        - name: GraphQL DGS Code Generation
+          id: dgs-codegen
+          compatibilityRange: "3.0.0-M1"
+          groupId: com.netflix.graphql.dgs.codegen
+          artifactId: graphql-dgs-codegen-gradle
+          mappings:
```

Comment:
Wondering if we need a mapping here considering that the artifact is just a trigger for the build system setup.

Reply:
Right now the only added benefit is that if we upgrade the plugin version and it's not fully published on Maven Central, our verification tests will catch it. I'd be tempted to leave that in for this reason.